extensions/meme_stats.py

```diff
@@ -0,0 +1,59 @@
+"""
+Displays a graph of meme ratings for a user, grouped by day.
+Todo: make day/hour/week whatever a parameter.
+"""
+import db
+import hikari
+import lightbulb
+import matplotlib.font_manager as fm
+import matplotlib.image as image
+import matplotlib.pyplot as plt
+from io import BytesIO
+
+plugin = lightbulb.Plugin("MemeStats")
+
+@plugin.command
+@lightbulb.add_cooldown(10, 1, lightbulb.UserBucket)
+@lightbulb.option("target", "The member to roast (or not?).", hikari.User, required=True)
+@lightbulb.command("memestats", "See someone's average meme rating over time")
+@lightbulb.implements(lightbulb.PrefixCommand,lightbulb.SlashCommand)
+async def main(ctx: lightbulb.Context) -> None:
+    guild = ctx.get_guild()
+    cursor = db.cursor()
+    user_id = ctx.options.target.id
+    user = guild.get_member(user_id)
+
+    # todo: add a limit or aggregation of greater time periods or whatever
+    # todo: get someone else to do this
+    data = cursor.execute("""
```

Comment:
This will have group gaps where users don't post every day. I think that will make the graph look odd unless you're a Jimmy-level poster

Reply:
Good point, but if we're only interested in updates it might be tolerable? It needs more thought in general so this is more the MVP.

Reply:
@jimbxb it kinda does what you suggested now. Missing dates seem to leave a gap and is imo better than a 0.

Reply:
Yeah, whoops. I replied to the wrong thing. Thanks